crates/iota-types/src/base_types.rs

```diff
@@ -132,6 +132,9 @@ pub trait ConciseableName<'a> {
 
 #[serde_as]
 #[derive(Eq, PartialEq, Clone, Copy, PartialOrd, Ord, Hash, Serialize, Deserialize, JsonSchema)]
+/// `Storage ID` of any shared Object
+/// 
+/// The ID by which the object can be identified in global storage.
```

Comment:
Storage ID as a specific context from which the object ID alone may be unrelated. I'd keep that out from the description of this type.
```suggestion
/// The ID by which the object can be identified in global storage.
```

Reply:
Fixed in: 859e18f20cc0c938824bd54b1fc5524b77a37aae